AnnService/inc/SSDServing/SSDIndex.h

```diff
@@ -134,7 +134,6 @@ namespace SPTAG {
                             double endTime = threadws.getElapsedMs();
                             p_index->DebugSearchDiskIndex(p_results[index], p_internalResultNum, p_internalResultNum, &(p_stats[index]));
                             double exEndTime = threadws.getElapsedMs();
-                            p_results[index].ClearTmp();
```

Comment:
In the SetTarget, you have already allocated the memory for all quantized targets. So we deallocate the memory does not reduce the total memory usage?

Reply:
When vectorset is provided, after the search is complete we will load the full-dim vectors into memory. With ClearTmp feature, quantized query memory will have been freed. Without it, we need the memory for both ADC tables and the reranking vectors. 

Alternately, we can do the free manually after in a loop after `SearchSequential`, instead of inside `SearchSequential` logic